components/auth/RegisterInputName.tsx

```diff
@@ -0,0 +1,136 @@
+import { useState, useCallback, FC, KeyboardEvent } from 'react';
+import styled from 'styled-components';
+import Section from '@/components/Section';
+import { media } from '@/utils/theme';
+import TextInput from '@/components/TextInput';
+import Button from '@/components/Button';
+
+import Loading from '../Loading';
+
+const StyledSection = styled(Section)`
+  position: absolute;
+  left: 0;
+  top: 0;
+  height: 100vh;
+  > img {
+    position: absolute;
+    left: 0;
+    top: 0;
+    width: 100%;
+    height: 100%;
+  }
+`;
+
+const RegisterWrapper = styled.div`
+  position: relative;
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+  width: 100%;
+  height: 100%;
+  ${media('desktop')} {
+    width: 100%;
+  }
+`;
+
+const StyledTextInput = styled(TextInput)`
+  && {
+    width: 50%;
+    margin-bottom: 60px;
+  }
+`;
+
+const StyledButton = styled(Button)`
+  padding: ;
+  margin-top: 2em;
+`;
+
+const InfoWrapper = styled.div`
+  max-width: 1000px;
+  width: 100%;
+  padding: 40px 40px 72px 40px;
+  background-color: #ffffff;
+  font-weight: bold;
+  > p {
+    color: #a80100;
+    font-size: 4vmin;
+    letter-spacing: 1.7px;
+    display: block;
+    max-width: 397px;
+    width: 100%;
+    margin-bottom: 100px;
+    ${media('desktop')} {
+      font-size: 24px;
+    }
+  }
+`;
+
+export enum RegisterPageState {
+  RegisterSignIn = 'REGISTER_SIGNIN',
+}
+
+type TProps = {
+  setPageState: any;
+  setUserName: any;
+  userName: any;
+  initUserName: (e: any) => void;
+  onChangePageState: (e: any) => void;
+  onChangeUserName: (e: any) => void;
+}
+
+const inputName: FC<TProps> = (props) => {
```

Comment:
A component name should be start with `capital` like `InputName`.

Or it will throw this warning~~
![image](https://user-images.githubusercontent.com/14778824/69305589-828e1f00-0c5f-11ea-9892-723bac7c8bce.png)